inductiva/examples/examples.py

```diff
@@ -0,0 +1,102 @@
+"""Functions to download and load examples from URLs.
+
+It contains several samples of molecules and a test_vehicle."""
+import os
+from typing import Optional
+from urllib import request, error
+
+from absl import logging
+
+import inductiva
+
+
+def load_insulin(save_dir: Optional[str] = None) -> str:
```

Comment:
There are many problems with this method. 
One is the naming, which obviously does not match the download_from_rcbp

Why would you even name a method *load* when what it does is *download* a file to a certain directory??
It also seems that we don't need to have these methods at all. The method   download_from_rcsb seems to be more than enough to illustrate our examples. It's actually perfect. Don't overcomplicate. Just use download_from_rcsb which is perfectly clear.

Reply:
So you think we should remove the functions to download specific examples from RCSB ? For that the user needs to know the exact PDB ID for the protein. The wrapper would make the example super clean and a user could load a molecule without knowing anything about PDB IDs.